<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page_graphic</class>
 <widget class="QWidget" name="Page_graphic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1222</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image_7">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>20</width>
     <height>620</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>方周长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_28">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_29">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_32">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_26">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_22">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>235</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_46">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>空白周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_47">
   <property name="geometry">
    <rect>
     <x>1005</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>空白周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_48">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_21">
   <property name="geometry">
    <rect>
     <x>375</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_49">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>255</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_50">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小边长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>方周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_51">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_23">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_17">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>112</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_15">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_16">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>225</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="image_6">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_5">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="image_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>这是 图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>方面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_24">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>235</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="image_5">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_11">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_52">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>205</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_19">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>225</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_53">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>外圆内方</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_54">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>半径：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_34">
   <property name="geometry">
    <rect>
     <x>1065</x>
     <y>85</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_55">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>方面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_25">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_56">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>240</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆环面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_57">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_58">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>空白面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_59">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>255</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_60">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_14">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>35</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="image_3">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_61">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_62">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>205</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>大边长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_63">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>205</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_27">
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_20">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>250</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_30">
   <property name="geometry">
    <rect>
     <x>925</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_18">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_64">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>外方内圆</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_65">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>小边长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_66">
   <property name="geometry">
    <rect>
     <x>1005</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>空白面积：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_67">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>115</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_31">
   <property name="geometry">
    <rect>
     <x>795</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_33">
   <property name="geometry">
    <rect>
     <x>1065</x>
     <y>110</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="graphic_lE_9">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="image_8">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_68">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>240</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_4">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>150</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>10</y>
     <width>20</width>
     <height>621</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_69">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>112</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_70">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_71">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>65</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>圆周长：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_72">
   <property name="geometry">
    <rect>
     <x>865</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>阴影面积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>10</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>这是 图片</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_73">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>205</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>大边长：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
